packages/contracts/src/EmailAccountRecovery.sol

```diff
@@ -16,6 +16,9 @@ abstract contract EmailAccountRecovery {
     address public dkimAddr;
     address public emailAuthImplementationAddr;
 
+    mapping(address account => string[][]) public acceptanceSubjectTemplates;
```

Comment:
These could also be stored in a registry. See https://github.com/zkemail/email-recovery/pull/3

Need to think about how these are stored a bit more.